index.js

```diff
@@ -138,3 +138,41 @@ newSightingForm.addEventListener('submit', function (event) {
 /************************** EVENTS PART 2 JS MINI CHALLENGE ******************************/
 
 
+//Deliv 1 & 2
+
+const dataIdLi = document.querySelector('ul#animals')
```

Comment:
This variable name is a bit confusing. I'm expecting it to be an LI but it is actually a UL. I would name it something like `animalCollectionUL` or `animalsUL`